IM_calculation/Advanced_IM/runlibs_2d.py

```diff
@@ -89,7 +92,7 @@ def main(args, im_name, run_script):
                 f"{getattr(args,component.str_value)} failed to converged in previous run."
             )
             model_converged = False
-            break
+            continue
```

Comment:
I think you have to update the comment and clarify what the code is trying to do here

Reply:
about the int part, i've updated the code base on james comment.

Reply:
So that. That will work.